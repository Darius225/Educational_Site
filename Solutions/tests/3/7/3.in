139
1 26003
2
2
0
1 8291
1 21932
2
0
0
1 17201
0
1 4873
2
1 29265
1 12258
1 2860
1 4092
0
0
1 22851
2
0
1 17596
2
1 30096
1 5788
1 14086
0
2
0
2
2
2
2
1 15140
0
1 3365
1 21217
1 16856
1 259
2
1 32174
0
0
1 31274
0
1 17599
2
0
0
1 6198
2
1 25605
0
0
2
2
0
0
1 9744
2
2
1 7302
0
0
2
2
0
2
0
1 11203
0
1 3593
0
2
1 1446
0
0
1 13762
1 32142
2
1 26833
2
2
2
0
0
0
0
0
1 26340
0
1 18298
0
1 9072
1 18314
0
1 26690
1 31994
0
1 13056
2
0
2
1 11820
2
1 23814
2
1 19404
0
2
2
2
0
2
0
0
1 22220
0
1 22031
1 25170
0
2
0
1 8822
0
2
1 29412
0
2
2
1 21737
2
0
1 26775
0
1 3122
0
0
