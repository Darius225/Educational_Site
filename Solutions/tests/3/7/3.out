26003
26003
21932
4873
22851
17596
5788
30096
30096
30096
30096
259
17599
6198
21217
21217
9744
9744
30096
30096
17596
12258
32142
26833
26833
26833
13056
26690
11820
23814
23814
23814
23814
11820
22031
9072
9072
9072
21737
